build-a-game-playing-agent/sample_players.py

```diff
@@ -243,8 +243,8 @@ def get_move(self, game, legal_moves, time_left):
 
                 if not valid_choice:
                     print('Illegal move! Try again.')
+                    continue
```

Comment:
oh, right, you're correct.  That is a mistake.  This would result in an infinite loop because `valid_choose` is statically False.  I agree this requires further review.